charts/member-agent/templates/deployment.yaml

```diff
@@ -105,6 +113,11 @@ spec:
       volumes:
```

Comment:
add `{{- if or (not .Values.useCAAuth) (eq .Values.propertyProvider "azure") }}` before this line